src/scripts/main.js

```diff
@@ -1,3 +1,271 @@
+
+  label.append(input);
+
+  return label;
+}
+
+const nameInput = createLabeledInput('Name:', 'text', 'name', 'name');
+const positionInput = createLabeledInput(
+  'Position:',
+  'text',
+  'position',
+  'position',
+);
+const ageInput = createLabeledInput('Age:', 'number', 'age', 'age');
+const salaryInput = createLabeledInput('Salary:', 'number', 'salary', 'salary');
+
+const officeLabel = document.createElement('label');
+const officeSelect = document.createElement('select');
+
+officeLabel.textContent = 'Office:';
+officeSelect.name = 'office';
+officeSelect.dataset.qa = 'office';
+officeSelect.required = true;
+
+const offices = [
+  'Tokyo',
+  'Singapore',
+  'London',
+  'New York',
+  'Edinburgh',
+  'San Francisco',
+];
+
+offices.forEach((office) => {
+  const option = document.createElement('option');
+
+  option.value = office;
+  option.textContent = office;
+  officeSelect.append(option);
+});
+officeLabel.append(officeSelect);
+
+const saveButton = document.createElement('button');
+
+saveButton.type = 'submit';
+saveButton.textContent = 'Save to table';
+
+form.append(nameInput);
+form.append(positionInput);
+form.append(officeLabel);
+form.append(ageInput);
+form.append(salaryInput);
+form.append(saveButton);
+
+document.body.append(form);
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+
+  const formData = new FormData(form);
+
+  const newPerson = {
+    Name: formData.get('name'),
+    Position: formData.get('position'),
+    Office: formData.get('office'),
+    Age: formData.get('age'),
+    Salary: `$${Number(formData.get('salary')).toLocaleString()}`,
+  };
+
+  const nameIsValid = newPerson.Name.length >= 4;
+  const ageIsValid = newPerson.Age >= 18 && newPerson.Age <= 90;
+  const positionIsValid = !!newPerson.Position;
+
+  if (!positionIsValid) {
+    const error = document.createElement('div');
+    const title = document.createElement('h1');
+    const message = document.createElement('p');
+
+    title.classList.add('title');
+    error.classList.add('notification', 'error');
+    error.setAttribute('data-qa', 'notification');
+
+    title.textContent = 'Position Error!';
+    message.textContent = 'Position must be entered.';
+
+    error.append(title);
+    error.append(message);
+    document.documentElement.append(error);
+
+    setTimeout(() => {
+      error.remove();
+    }, 2000);
+
+    return;
+  }
+
+  if (nameIsValid && ageIsValid) {
+    const success = document.createElement('div');
+    const title = document.createElement('h1');
+    const message = document.createElement('p');
+
+    title.classList.add('title');
+    success.classList.add('notification', 'success');
+    success.setAttribute('data-qa', 'notification');
+
+    title.textContent = 'Submission Successful!';
+    message.textContent = 'Your information has been added successfully.';
+
+    success.append(title);
+    success.append(message);
+    document.documentElement.append(success);
```

Comment:
Appending the success notification directly to `document.documentElement` might not be the best approach. Consider appending it to a specific container or the body to maintain a consistent structure.